powerauth-push-server/src/main/java/io/getlime/push/configuration/StorageConfiguration.java

```diff
@@ -16,25 +16,32 @@
 
 package io.getlime.push.configuration;
 
-import io.getlime.push.service.batch.storage.AppCredentialStorageMap;
+import io.getlime.push.service.batch.storage.AppCredentialStorage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.time.Duration;
+
 
 /**
- * Configuration class for in memory maps
+ * Configuration class for storage.
  *
  * @author Martin Tupy, martin.tupy.work@gmail.com
  */
 @Configuration
-public class StorageMapConfiguration {
+@Slf4j
+public class StorageConfiguration {
```

Comment:
I would rather call it `PushServiceConfiguration` but there already is such a class, which should be named `PushServiceConfigurationProperties`. Anyway, let's postpone to another PR.